core/domain/state_domain.py

```diff
@@ -3364,6 +3392,7 @@ def convert_html_fields_in_state(
             state_dict: dict. The dict representation of State object.
             conversion_fn: function. The conversion function to be applied on
                 the states_dict.
+            state_schema_version: number. The state schema version.
```

Comment:
Shouldn't "number" be "int"?

Reply:
Done in https://github.com/oppia/oppia/pull/14371.